indy-utils/src/main/kotlin/com/luxoft/blockchainlab/hyperledger/indy/ledger/IndyPoolLedgerUser.kt

```diff
@@ -83,7 +94,7 @@ class IndyPoolLedgerUser(val pool: Pool, val did: String, val signProvider: (dat
 
             true
         } catch (e: Exception) {
-            logger.error { e }
+            logger.debug { e }
```

Comment:
Why?

Reply:
Because this is not error, but usual life cycle.